core/server/overrides.js

```diff
@@ -1,4 +1,5 @@
 const moment = require('moment-timezone');
+const {extract, hasProvider} = require('oembed-parser'); // eslint-disable-line
```

Comment:
Could you please leave a comment here why we have to require the lib here? Thanks 👍